frontend/tests/project-creation.spec.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { test, expect } from '@playwright/test';
+
+const nearPixelArea = 50; // Specifies how far away an element can be when using near
+
+test('test the full project creation workflow', async ({ page }) => {
+  const testProjectName = `test-project-name-${Math.trunc(
+    Math.random() * 1000
+  )}`;
+  const testProjectDescription = 'test-project-description';
+
+  await page.goto('http://localhost:4200/');
+
+  // Check whether you got redirected to the /auth page
+  await expect(page).toHaveURL('http://localhost:4200/auth');
+
+  await page.locator('button:text-matches("Login with .*", "g")').click();
```

Comment:
This line failed for me, because the "RELEASE NOTES" dialog was popping up. I've reworked the dialog so that it doesn't popup automatically: https://github.com/DSD-DBS/capella-collab-manager/pull/288